spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -808,9 +808,15 @@ def change_clients_pdb_use_exclamation_mark(self, value):
     @on_conf_change(option=[
         'symbolic_math', 'hide_cmd_windows',
         'startup/run_lines', 'startup/use_run_file', 'startup/run_file',
-        'pylab', 'pylab/backend'])
-    def change_possible_restart_conf(self, option, value):
-        """Apply options that possibly require a kernel restart."""
+        'pylab', 'pylab/backend',
+        'pylab/autoload', 'pylab/inline/figure_format',
+        'pylab/inline/resolution', 'pylab/inline/width',
+        'pylab/inline/height', 'pylab/inline/bbox_inches'])
```

Comment:
```suggestion
        'pylab', 'pylab/backend', 'pylab/autoload',
        'pylab/inline/figure_format', 'pylab/inline/resolution',
        'pylab/inline/width', 'pylab/inline/height',
        'pylab/inline/bbox_inches'])
```

Make options fit better in 80 column width